src/ruleLoader.js

```diff
@@ -0,0 +1,10 @@
+module.exports = function () {
+  const callback = this.async();
+  const { plugin } = this.query;
+
+  plugin.tags.promise
+    .then(tags => {
+      callback(null, "module.exports = ['" + tags.join( "', '" ) + "'];");
+    })
+    .catch(callback);
+};
```

Comment:
Would you mind renaming this to `rule_loader.js` to avoid issues with case insensitive file systems?

Reply:
Done